src/lib/y2storage/autoinst_issues/multiple_bcache_members.rb

```diff
@@ -0,0 +1,67 @@
+# encoding: utf-8
+
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/autoinst_issues/issue"
+
+module Y2Storage
+  module AutoinstIssues
+    # The proposal several backing or caching devices for the same bcache
+    #
+    # This is a fatal error.
+    class MultipleBcacheMembers < Issue
+      # @return [String] Bcache member role (:backing or :caching)
+      attr_reader :role
+      # @return [String] Bcache device name
+      attr_reader :bcache_name
+
+      # Constructor
+      #
+      # @param role        [Symbol] :backing or :caching
+      # @param bcache_name [String] Bcache device name
+      def initialize(role, bcache_name)
+        super()
+        textdomain "storage"
+
+        @role = role
+        @bcache_name = bcache_name
+      end
+
+      # Return problem severity
+      #
+      # @return [Symbol] :fatal
+      def severity
+        :fatal
+      end
+
+      # Return the error message to be displayed
+      #
+      # @return [String] Error message
+      # @see Issue#message
+      def message
+        # TRANSLATORS: 'bcache_name is the bcache device name (e.g., '/dev/bcache0');
+        # 'role' is the kind of device (e.g., 'caching' or 'backing').
+        _(
+          "%{bcache_name}: only one %{role} device can be specified per bcache."
```

Comment:
Not sure how nice is to have the untranslated word "backing" or "caching" in all messages.

E.g.: "/dev/bcache0: sólo puede especificarse un dispositivo backing para cada bcache"

Well, the profile of course use English names, so I guess that's acceptable.

Reply:
You are right. I've decided to make "backing device" and "caching device" as the strings to translate.